packages/flutter/test/rendering/view_chrome_style_test.dart

```diff
@@ -16,18 +14,17 @@ void main() {
     const double deviceWidth = 480.0;
     const double devicePixelRatio = 2.0;
 
-    void setupTestDevice() {
-      final TestWidgetsFlutterBinding binding = TestWidgetsFlutterBinding.ensureInitialized();
+    void setupTestDevice(WidgetTester tester) {
       const FakeViewPadding padding = FakeViewPadding(
         top: statusBarHeight * devicePixelRatio,
         bottom: navigationBarHeight * devicePixelRatio,
       );
 
-      binding.window
-        ..viewPaddingTestValue = padding
-        ..paddingTestValue = padding
-        ..devicePixelRatioTestValue = devicePixelRatio
-        ..physicalSizeTestValue = const Size(
+      tester.view
```

Comment:
Should we add a reset?

Reply:
Very good call. Added an `addTearDown` to the setup method.